src/EventStore.Core/Util/Opts.cs

```diff
@@ -259,14 +259,14 @@ public static class Opts {
 		public const string GossipTimeoutMsDescr = "The timeout, in ms, on gossip to another node.";
 		public const int GossipTimeoutMsDefault = 2500;
 
-		public const string AdminOnExtDescr = "Whether or not to run the admin ui on the external HTTP endpoint";
+		public const string AdminOnExtDescr = "Whether or not to run the admin ui on the HTTP endpoint";
```

Comment:
@hayley-jean @pgermishuys Do we need to rename these three options: `AdminOnExt`, `GossipOnExt` and `StatsOnExt` ? (since the `OnExt` suffix is no longer applicable)

Reply:
I think that `GossipOnExt` can be removed altogether since we should not be able to run a cluster if it's set to false.
`AdminOnExt` could be renamed to `DisableAdminUI` or something similar and `StatsOnExt` renamed to `DisableHttpStats`

Reply:
`GossipOnExt` could still be used as it could be used to disable other clients from connecting to the gossip (such as the TCP or grpc client) as these still use http gossip, not grpc.

Renaming the options is something we could do, so:
`DisableAdminUI`
`DisableStatsOnHttp`
`DisableGossipOnHttp`

Reply:
@hayley-jean thanks, yup I missed this point regarding GossipOnExt.